src/System.Device.Gpio/System/Device/Gpio/BoardIdentification .cs

```diff
@@ -0,0 +1,337 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace System.Device.Gpio
+{
+    /// <summary>
+    /// Identification of Raspberry Pi board models
+    /// </summary>
+    internal class BoardIdentification
```

Comment:
Yes, I should change the name to become raspi-specific. 

This is on purpose internal for now, as krwq insists that we currently do not add new public apis to the core dll. Once we agree on the way forward with board identification and hardware abstraction layers, this will likely change again.